clash-lib/src/Clash/Netlist/Types.hs

```diff
@@ -498,6 +499,10 @@ data PortMap
   --
   deriving (Show)
 
+-- | @component@ declaration
+data CompDecl
+  = VHDLComp !Text [(Text,PortDirection,HWType)]
```

Comment:
Use spaces after `,`. We do do this around the Clash code base, but it's a relic of the past :).

Reply:
Call the constructor after the type name too

Reply:
Oh I see!